docs/settings/monitoring-settings.asciidoc

```diff
@@ -45,13 +45,13 @@ production cluster as well as monitor data sent to a dedicated monitoring
 cluster.
 
 `xpack.monitoring.elasticsearch.username`::
-Specifies the user ID that {kib} uses for authentication when it retrieves data
-from the monitoring cluster. If not set, {kib} uses the value of the
+Specifies the user ID that {kib} uses for authentication when it sends data
```

Comment:
This isn't entirely accurate. I had some [revisions on this comment](https://github.com/elastic/sdh-kibana/issues/73#issuecomment-540640413), but it's now up to date and reflects the reality of what we're doing.

This user id is only used for two things:

- Used to authenticate against the _monitoring_ cluster in order to establish the persistent connection (which we later use to read data from, but using different credentials)
- Used to fetch xpack info data from the _monitoring_ cluster in order to determine licensing and product availability (of the monitoring cluster)

Other than that, it's not used for anything else in monitoring. All reads are done against the _monitoring_ cluster but with the authenticated user credentials.

Reply:
Gotcha. I was going off the prior revision when I changed this.

Given this, I'm struggling to come up with a use case for _ever_ setting these options.

At first, I thought there might be a case where one might need to hit the `/_xpack`API and wouldn't be able to, but I verified that any user with the `monitoring_user` role can hit this API and get a full set of results.

Given that all the reads on the persistent connection are performed as the logged-in user, is there a specific reason that the connection establishment needs to be as another user? I'm not too familiar with the ES connection establishment code inside Kibana.

Reply:
I'm assuming for the same reasons we even need a `elasticseach.username` and `elasticsearch.password`, which honestly I don't quite know offhand. I imagine it handles cases where Kibana server is running, but there isn't a user logged in using the client. Perhaps we don't necessarily need that for monitoring though.

Reply:
@chrisronline I've updated the copy. Would you mind please taking another look?